tests/Localization/CkbTest.php

```diff
@@ -124,108 +124,108 @@ class CkbTest extends LocalizationTestCase
         // Carbon::parse('2018-01-01 00:00:00')->ordinal('hour')
         '0',
         // Carbon::now()->subSeconds(1)->diffForHumans()
-        'پێشتر چرکە',
+        'پێش چرکەیەک',
```

Comment:
```suggestion
        'پێشتر چرکەیەک',
```

Reply:
In the latest changes, this problem must have been fixed.
How can I run this test, to make sure the words are correct?